packages/dashboard/src/components/iot-dashboard/iot-dashboard-messages/messages.ts

```diff
@@ -0,0 +1,47 @@
+export type ContextMenuMessages = {
+  copyText: string;
+  pasteText: string;
+  deleteText: string;
+  bringToFrontText: string;
+  sendToBackText: string;
+};
+
+export type WidgetsMessages = {
+  invalidTagHeader: string;
+  invalidTagSubheader: string;
+};
+
+export type DashboardMessages = {
+  widgets: WidgetsMessages;
+  contextMenu: ContextMenuMessages;
+};
+
+export type DashboardMessageOverrides = {
+  widgets: Partial<WidgetsMessages>;
+  contextMenu: Partial<ContextMenuMessages>;
+};
+
+export const DefaultDashboardMessages: DashboardMessages = {
+  widgets: {
+    invalidTagHeader: 'Widget failed to load',
+    invalidTagSubheader: 'Please try again later or contact an admin for help.',
+  },
+  contextMenu: {
+    copyText: 'Copy',
+    pasteText: 'Paste',
+    deleteText: 'Delete',
+    bringToFrontText: 'Bring to Front',
+    sendToBackText: 'Send to Back',
+  },
+};
+
+export const IsMacLike = /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform);
```

Comment:
first letter capitalized should be reserved for class names and typescript types (i.e. rename to `isMacLike`)

Reply:
👍